SkippedItem handling updates (#2485)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md

We recommend that the primary address (sometimes referred to as the "User ID") for each user be at the primary domain (such as "will@fabrikaminc.net"). Typically, this means that the primary email address should match between Microsoft 365 or Office 365 and Google Workspace. If any user is provisioned with a different domain for their primary address, then that user should at least have a proxy address at the primary domain. Each user should have their `ExternalEmailAddress` point to the user in their Google Workspace routing domain ("will@gsuite.fabrikaminc.net"). The users should also have a proxy address that will be used for routing to their Microsoft 365 or Office 365 routing domain (such as "will@o365.fabrikaminc.net").

>    > [!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

>    > It is recommended that MRM and Archive policies are disabled for these users until after the migration for those users has been completed.  When such features remain enabled during migration, there is a chance that some messages will end up being considered "missing" during content verification process.

Reviewer: ```suggestion
> We recommend that the Default MRM Policy and Archive policies be disabled for these users until after their migration has been completed.  When such features remain enabled during migration, there is a chance that some messages will end up being considered "missing" during the content verification process.
```
